responsibleai/responsibleai/managers/error_analysis_manager.py

```diff
@@ -165,28 +152,11 @@ def from_json(json_str):
 
 
 class ErrorAnalysisManager(BaseManager):
-
-    """Defines the ErrorAnalysisManager for discovering errors in a model.
-
-    :param model: The model to analyze errors on.
```

Comment:
This block is duplicated in the constructor, I assume?

Reply:
![image](https://user-images.githubusercontent.com/10245648/153276878-fc56fe61-c6ca-42de-b9fd-4408f8007c99.png)

Yes!